README.md

```diff
@@ -50,7 +50,7 @@ Looking for help? Check out the
   April and October every year. Releases appearing each October have a support
   life of 8 months. Releases appearing each April convert to LTS (see below)
   each October.
-* **LTS**: Releases that receive Long-term Support, with a focus on stability
+* **LTS**: Releases that receive Long-Term Support, with a focus on stability
```

Comment:
Per https://github.com/nodejs/Release it should really not have a hypen at all:
```suggestion
* **LTS**: Releases that receive Long Term Support, with a focus on stability
```

Reply:
Yeah I agree. When searching for "Node.js Long-Term Support", it seems that the convention is "Long Term Support" as a title and "long-term support" when written in sentences.